src/main/java/org/broadinstitute/consent/http/service/SupportRequestService.java

```diff
@@ -133,40 +141,59 @@ public void handleSuggestedUserFieldsSupportRequest(UserUpdateFields userUpdateF
-            description = String.format("User %s [%s] has requested a new institution: {%s}",
-                    user.getDisplayName(),
-                    user.getEmail(),
-                    suggestedInstitution);
-        } else if (Objects.nonNull(suggestedSigningOfficial)) {
-            subject = "New Signing Official Request";
-            description = String.format("User %s [%s] has requested a new signing official: {%s}",
-                    user.getDisplayName(),
-                    user.getEmail(),
-                    suggestedSigningOfficial);
+        Integer selectedSigningOfficialId = userUpdateFields.getSelectedSigningOfficialId();
+
+        //Generate subject and description of ticket based on provided fields
+        List<String> subjectItems = new ArrayList<>();
+        List<String> descriptionItems = new ArrayList<>();
+        if (Objects.nonNull(suggestedInstitution)) {
+            subjectItems.add("New Institution Request");
+            descriptionItems.add("- requested a new institution: " + suggestedInstitution);
+        }
+        if (Objects.nonNull(selectedInstitutionId)) {
+            Institution selectedInstitution = institutionDAO.findInstitutionById(selectedInstitutionId);
+            subjectItems.add("Institution Selection");
+            descriptionItems.add(
+                    Objects.nonNull(selectedInstitution)
+                            ? "- selected an existing institution: " + selectedInstitution.getName()
+                            : "- attempted to select institution with id " + selectedInstitutionId + " (not found)"
+            );
+        }
+        if (Objects.nonNull(suggestedSigningOfficial)) {
+            subjectItems.add("New Signing Official Request");
+            descriptionItems.add("- requested a new signing official: " + suggestedSigningOfficial);
+        }
+        if (Objects.nonNull(selectedSigningOfficialId)) {
+            User selectedSigningOfficial = userDAO.findUserById(selectedSigningOfficialId);
+            subjectItems.add("Signing Official Selection");
+            descriptionItems.add(
+                    Objects.nonNull(selectedSigningOfficial)
+                            ? String.format("- selected an existing signing official: %s, %s",
+                            selectedSigningOfficial.getDisplayName(),
+                            selectedSigningOfficial.getEmail())
+                            : "- attempted to select signing official with id " + selectedSigningOfficialId + " (not found)"
+            );
         }
 
+        //Append items for ticket subject and description
+        String subject = String.format("%s user updates: %s",
+                user.getDisplayName(),
+                String.join(", ", subjectItems));
+        String description = String.format("User %s [%s] has:\n%s",
+                user.getDisplayName(),
+                user.getEmail(),
+                String.join("\n", descriptionItems));
```

Comment:
There's a lot of code here in the service class that is very specific to ticket creation/population. An optional pattern would be to extract a chunk of that logic into a model class so it's more encapsulated. Not a requirement, just a style suggestion. See how we use generic utility methods on `UserUpdateFields` as a possible example.